config/routes.rb

```diff
@@ -3,13 +3,15 @@
   mount JasmineRails::Engine => '/specs' if defined?(JasmineRails)
   mount JasmineFixtureServer => '/spec/javascripts/fixtures' if defined?(Jasmine::Jquery::Rails::Engine)
 
+  # Manually written API functions
+  post '/bot/comment/id.:format', to: 'comment#create_by_token'
```

Comment:
i'd prefer this to be more general -- anyone developing an API use -- could we do `/comment/create/token/:id.:format` maybe?